frontend/packages/dev-console/integration-tests/features/login.feature

```diff
@@ -0,0 +1,30 @@
+Feature: Login to Application
+
+    As a valid user
+    I want to log in to the Application
+
+    @smoke, @odc-1255
+    Scenario Outline: Valid Login for "<userName>" : P-02-TC01
+        Given I open the login Page
+        When I enter a username "<userName>"
```

Comment:
Can you please update it with the "User" and not "I"

Reply:
I will update